src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../styles/states.scss";
+@import "../../styles/global-css-settings.scss";
+
+@mixin toggle-animation {
+  transition: 100ms $expand-animation-timing;
```

Comment:
not related to this cr specifically but I see the project use 100ms a lot, maybe worth to keep it as a variable together with $expand-animation-timing ?

Reply:
@orrgottlieb ?

Reply:
i would add times contants - like fast - 50ms, medium - 100ms, slow - 250ms